### PR TITLE
Profiles: make hidden nfts work with profiles

### PR DESCRIPTION
**Pull request by jxom**
Fixes TEAM2-389

## What changed (plus any additional context for devs)

This PR makes Hidden NFTs work with profiles. Since we don't use redux for data fetching on the profiles screen, need to add interop. Obviously having duplicate code in 2 diff places is not good, but we will need to refactor this in Code Foundations to be universal.

## Screen recordings / screenshots

https://user-images.githubusercontent.com/7336481/183329744-cf4a3386-625e-4636-84c7-055ea80fcae5.mp4

## What to test

Hide an NFT in a wallet, and ensure that it is hidden on the profiles screen.

### CODE REVIEW

File: src/helpers/assets.ts [hunk 1]
```diff
@@ -414,7 +415,7 @@ export const buildBriefUniqueTokenList = (
 
     result.push({ type: 'NFT_SPACE_AFTER', uid: `${family}-space-after` });
   }
-  if (hiddenUniqueTokensIds.length > 0) {
+  if (hiddenUniqueTokensIds.length > 0 && listType === 'wallet') {
```
Comment (jxom) [inline src/helpers/assets.ts:418]: fyi, the other list types are `"ens-profile"` & `"select-nft"`, where we do not want to show hidden NFTs.

Comment (skylarbarrera) [reply src/helpers/assets.ts:418]: yesh

**Review by skylarbarrera — APPROVED** [review]
lgtm

**Review by estebanmino — APPROVED** [review]
i've been hiding nfts on android and ios, the checking profiles, all looking nice 💯

**Pull request merged**